packages/components/filter-bar/filter-bar-refresher.ts

```diff
@@ -0,0 +1,27 @@
+import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import { KbqButtonModule } from '@koobiq/components/button';
+import { KbqIconModule } from '@koobiq/components/icon';
+
+@Component({
+    standalone: true,
+    selector: 'kbq-filter-bar-refresher, [kbq-filter-bar-refresher]',
```

Comment:
а зачем здесь селектор директивы? кажется что это явный компонент
```suggestion
    selector: 'kbq-filter-bar-refresher, [kbq-filter-bar-refresher]',
```

Reply:
на рефрешер забили, позже будет дизайн